### ISSUE TITLE
Upstream: a11y(critical) frontend-base alt and link name issues on shell header/footer

### ISSUE THREAD

**Issue by bmtcril** [openedx/end-to-end-tests#31]
Right now we are globally ignoring critical a11y `image-alt` and `link-name` violations since they show up on the header and footer of every frontend-base page. Once this is fixed we will start to fail on those _not_ violating and can remove the global exclusion.

Upstream issue: https://github.com/openedx/frontend-base/issues/314
